<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1866</width>
    <height>993</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Alignmet System</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_img1">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>36</y>
      <width>750</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);border-width: 1.2px; border-style: inset</string>
    </property>
    <property name="text">
     <string>Cam1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_img2">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>36</y>
      <width>750</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);border-width: 1.2px; border-style: inset</string>
    </property>
    <property name="text">
     <string>Cam2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>340</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>MANUAL DETECT</string>
    </property>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Normal Direection Detecti</string>
     </property>
     <widget class="QPushButton" name="Calculate_Button">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>76</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Calculate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Find_Contour_Button">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>50</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>2.Find Contour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Image_Load_Button">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>24</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1.Image Load</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Show_Graph_Button">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>102</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>4.Show Graph</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>127</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>127</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>930</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Gradient (Gx[Blue], Gy[Red])&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>640</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>930</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Angle&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>927</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Magnitude&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>640</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>340</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Detect position</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Left Camera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>36</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Left_x_detect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>36</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Left_y_detect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>51</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>63</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Right Camera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>97</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Right_x_detect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Right_y_detect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>96</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>450</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>G.T.</string>
    </property>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Left Camera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>35</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Left_x_GT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>36</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>54</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Left_y_GT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>54</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Right Camera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>88</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X: </string>
     </property>
    </widget>
    <widget class="QLabel" name="Right_x_GT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>88</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Right_y_GT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>106</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>570</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Difference(Detect&lt;-&gt;G.T.)</string>
    </property>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>21</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="diff_x_left">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>24</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="diff_y_left">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>23</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="diff_x_right">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>24</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="diff_y_right">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>47</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1866</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
